#Envia a cor para um LED RGB

int32 r #Intensidade de cor red do led (0 a 255)
int32 g #Intensidade de cor green do led (0 a 255)
int32 b #Intensidade de cor blue do led (0 a 255)